docs/30-customize-cluster.md

```diff
@@ -1,19 +1,127 @@
-# Running Commands when the Cluster Starts
+# Customizing the cluster
+
+There are several ways to control what gets deployed to a cluster. The most flexible and powerful method is to manage the docker conatiner image that is used to provision the cluster. By default doAzureParallel uses containers to provision the R environement. Anything within the container will be available on all nodes in the cluster. The default container used in doAzureParallel is [rocker/tidyverse:latest](https://hub.docker.com/r/rocker/tidyverse/) developed and maintained by the rocker organization.
+
+## Modifying the default docker container
+
+Specifying a docker container is done by updating your cluster.json file. Simply adding a reference to the docker container in the cluster.json file 'containerImage' property will cause all new clusters to deploy that conatiner to your cluster. doAzureParallel will use the version of R specified in the container.
+
+```json
+{
+  "name": "myPool",
+  "vmSize": "Standard_F2",
+  "maxTasksPerNode": 1,
+  "poolSize": {
+    "dedicatedNodes": {
+      "min": 0,
+      "max": 0
+    },
+    "lowPriorityNodes": {
+      "min": 1,
+      "max": 1
+    },
+    "autoscaleFormula": "QUEUE"
+  },
+  "containerImage": "rocker/tidyverse:3.4.1",
+  "rPackages": {
+    "cran": [],
+    "github": [],
+    "bioconductor": [],
+    "githubAuthenticationToken": ""
+  },
+  "commandLine": []
+}
+```
+
+Note: \_If no 'containerImage' property is set, rocker/tidyverse:latest will be used. This usually points to one of the latest versions of R.\_
+
+### Building your own container
+
+Building your own container gives you the flexibility to package any specific requirements, packages or data you require for running your workloads. We recommend using a debian based OS such as debian or ubuntu to build your containers and pointing to where R is in the final CMD command. For example:
+
+```dockerfile
+FROM debian:stretch
```

Comment:
maybe change one of these to `FROM rocker/tidyverse:latest`
thoughts?

Reply:
Yes, we can switch it out. I think that's fine rocker/tidyverse is built on top of debian:stretch anyways. Users might not need tidyverse though

Reply:
Agree with Brian - BUT we should have more samples for this.